packages/core/src/crawlers/error_snapshotter.ts

```diff
@@ -0,0 +1,135 @@
+import crypto from 'node:crypto';
+
+import type { ErrnoException } from './error_tracker';
+import type { CrawlingContext } from '../crawlers/crawler_commons';
+import type { KeyValueStore } from '../storages';
+import type { PlaywrightCrawlingContext, PuppeteerCrawlingContext, PlaywrightPage, PuppeteerPage } from '../typedefs';
+
+const { PWD, CRAWLEE_STORAGE_DIR, APIFY_IS_AT_HOME } = process.env;
```

Comment:
and i would discourage this completely, you explode the values on require time, this means there is no way to change the env var during runtime - which can be handy in tests, and not only there.

just use `process.env.XXX` directly, it's more readable anyway